src/Pos/Wallet/Web/Api.hs

```diff
@@ -365,39 +360,39 @@ type WalletApi = ApiPrefix :> (
      -------------------------------------------------------------------------
      -- Wallet sets
      -------------------------------------------------------------------------
-     GetWalletSet
+     GetWallet
     :<|>
-     GetWalletSets
+     GetWallets
     :<|>
-     NewWalletSet
+     NewWallet
     :<|>
-     RestoreWalletSet
+     RestoreWallet
     :<|>
-     RenameWalletSet
+     RenameWallet
     :<|>
-     DeleteWalletSet
+     DeleteWallet
     :<|>
-     ImportWalletSet
+     ImportWallet
     :<|>
-     ChangeWalletSetPassphrase
+     ChangeWalletPassphrase
     :<|>
      -------------------------------------------------------------------------
      -- Wallets
      -------------------------------------------------------------------------
-     GetWallet
+     GetAccount
     :<|>
-     GetWallets
+     GetAccounts
     :<|>
-     UpdateWallet
+     UpdateAccount
     :<|>
-     NewWallet
+     NewAccount
     :<|>
-     DeleteWallet
+     DeleteAccount
     :<|>
      -------------------------------------------------------------------------
      -- Accounts
```

Comment:
Lil, where did you get this commit? ))
One which Ante commented before already has comment fixed